x/oracle/keeper/keeper.go

```diff
@@ -69,6 +72,49 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 	return ctx.Logger().With("module", fmt.Sprintf("x/%s", types.ModuleName))
 }
 
+func (k Keeper) GetCurrentRoundInfo(ctx sdk.Context) *types.RoundInfo {
```

Comment:
Can't we derive `roundInfo` from current block height and params, instead of using the state db?

Reply:
I am wondering about the situation when the votePeriod is changed. However, this situation could be managed if we assign the roundId differently.(ex. random)

Reply:
I'll change the roundId format to uuid and change not to use statedb.

Reply:
if we use a random value like `uuid`, how do we pass that value to the next block without using the state db?

I think keeping `RoundInfo.id` as `uint64` is great! We can always query `x/oracle` module's param store and derive the `RoundInfo.id` even if we change votePeriod via governance.